scm/doc/TechGuide/chap_intro.rst

```diff
@@ -103,7 +103,7 @@ This release bundle has some known limitations:
 
 -  As of this release, using the SCM over a land point with an LSM is
    possible through the use of UFS initial conditions (see 
-   :numref:`Section %s <UFS ICs>`). However, advective forcing terms
+   :numref:`Section %s <UFSreplay>`). However, advective forcing terms
```

Comment:
FWIW. We are going to need to change all instances of "replay" in the documentation to "case generation".
There will be a subsequent PR with those code changes, so no need to address this at the moment.

Reply:
This link is currently broken (not caught in the last PR) so this just updates the link to point to the correct place.